app/src/main/java/org/simple/clinic/instantsearch/InstantSearchEffectHandler.kt

```diff
@@ -48,6 +48,7 @@ class InstantSearchEffectHandler @AssistedInject constructor(
       .addConsumer(OpenShortCodeSearchScreen::class.java, ::openShortCodeSearch, schedulers.ui())
       .addAction(OpenQrCodeScanner::class.java, uiActions::openQrCodeScanner, schedulers.ui())
       .addTransformer(CheckIfPatientAlreadyHasAnExistingNHID::class.java, checkIfPatientAlreadyHasAnExistingNHID(schedulers.ui()))
+      .addConsumer(ShowNHIDErrorDialog::class.java, { uiActions.showNHIDErrorDialog() }, schedulers.ui())
```

Comment:
This should be an action, not a consumer, because we don't use any properties from the effect.